### PR TITLE
Uni core p7 202007011200

### PR DESCRIPTION
**Pull request by jkeenan**
This p.r. addresses most warnings in `t/uni/*.t` observed once warnings are turned on by default.

There is one uninitialized value warning in `t/uni/parser.t` that I haven't yet been able to figure out.

There is still a tremendous amount of noise in `t/uni/variables.t` to be addressed.

Please read commit messages, where I've included caveats and TODOs.

Thank you very much.
Jim Keenan

### CODE REVIEW

File: t/uni/package.t [hunk 1]
```diff
@@ -36,8 +36,6 @@ our $TODO;
         $ㄅĽuṞfⳐ = 5;
     }
     
-    $압Ƈ'd읯ⱪ = 6;        #'
```
Comment (atoomic) [inline t/uni/package.t:39]: I think we should preserve this line, this was testing a string using single quote

File: t/uni/parser.t [hunk 2]
```diff
@@ -107,11 +107,14 @@ is grep({ $_ eq "\x{539f}"     } keys %::), 1, "Constant subs generate the right
 is grep({ $_ eq "\345\216\237" } keys %::), 0;
 
 #These should probably go elsewhere.
-eval q{ sub wròng1 :prototype(_$); wròng1(1,2) };
-like( $@, qr/Malformed prototype for main::wròng1/, 'Malformed prototype croak is clean.' );
+{
+    no warnings 'illegalproto';
```
Comment (atoomic) [inline t/uni/parser.t:111]: excellent

File: t/uni/readline.t [hunk 3]
```diff
@@ -6,24 +6,43 @@ BEGIN {
...
+        "Got expected number of 'unopened' warnings");
+}
 
 # [perl #21628]
 {
   my $file = tempfile();
   open Ạ,'+>',$file; $a = 3;
   is($a .= <Ạ>, 3, '#21628 - $a .= <A> , A eof');
   close Ạ; $a = 4;
+  #no warnings 'closed';
```
Comment (atoomic) [inline t/uni/readline.t]: useless line?

File: t/uni/universal.t [hunk 4]
```diff
@@ -103,8 +103,9 @@ ok $a->isa("Fèmàlè");
 ok (!Cèdrìc->isa('Prògràmmèr'));
 
 my $b = 'abc';
+our %c;
 my @refs = qw(SCALAR SCALAR     LVALUE      GLOB ARRAY HASH CODE);
-my @vals = (  \$b,   \3.14, \substr($b,1,1), \*b,  [],  {}, sub {} );
+my @vals = (  \$b,   \3.14, \substr($b,1,1), \*c,  [],  {}, sub {} );
```
Comment (atoomic) [inline t/uni/universal.t:108]: LGTM this does not change the test original purpose IMO

File: t/uni/universal.t [hunk 5]
```diff
@@ -141,6 +141,10 @@ package Pìg;
 package Bòdìnè;
 Bòdìnè->isa('Pìg');
 
+package zlòpp;
+
+package plòp;
+
```
Comment (atoomic) [inline t/uni/universal.t:147]: ok

**Review by atoomic — CHANGES_REQUESTED** [review]
my only concern is the removal from `t/uni/package.t`

File: t/uni/package.t (hunk 1 above)
Comment (jkeenan) [reply t/uni/package.t:39]: Restored.

File: t/uni/readline.t (hunk 3 above)
Comment (jkeenan) [reply t/uni/readline.t]: Done.

**Review by atoomic — APPROVED** [review]

**Pull request merged**